circus-train-aws/src/main/java/com/hotels/bdp/circustrain/aws/S3DataManipulator.java

```diff
@@ -54,6 +54,11 @@ public boolean delete(String path) {
     List<KeyVersion> keysToDelete = getKeysToDelete(bucket, uri.getKey());
     log.debug("Deleting keys: {}", keysToDelete.stream().map(k -> k.getKey()).collect(Collectors.toList()));
```

Comment:
Maybe move this to line 61 as there's no point outputting this if there's nothing to delete?

Reply:
yeah